### PR TITLE
Add bg-atlasapi recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by goanpeca** [comment]
Depends on:
- https://github.com/conda-forge/staged-recipes/pull/17712

**Comment by goanpeca** [comment]
We are in the process of creating conda packages for [napari plugins](https://www.napari-hub.org/) and related dependencies. Once this recipe passes all the checks, every time a new release goes into PyPI, a PR will be automatically open, with usually very minor need for changes so the process of maintaining this moving forward is small. Also the @conda-forge/napari team will be available to help as needed.

You can read a bit more about conda-forge at https://conda-forge.org/#contribute

--- 

@vigji, do you agree with being added as a maintainer?

File: recipes/bg-atlasapi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{% set name = "bg-atlasapi" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/brainglobe/bg-atlasapi/archive/v{{ version }}.tar.gz
```
Comment (goanpeca) [inline recipes/bg-atlasapi/meta.yaml:9]: The pypi dist was missing some files, so using the tagged github version.

Comment (jaimergp) [reply recipes/bg-atlasapi/meta.yaml:9]: As long as there is a Github **release**, yes. A tag is not enough. I think you could also technically list both sources (GH and PyPI), extracted to different folders, but only use the GH sources for the purposes of this recipe.

Comment (jaimergp) [reply recipes/bg-atlasapi/meta.yaml:9]: @vigji, can you confirm whether GH releases are expected for this project?

Comment (goanpeca) [reply recipes/bg-atlasapi/meta.yaml:9]: Pinging @vigji :)

Comment (vigji) [reply recipes/bg-atlasapi/meta.yaml:9]: Sorry I've missed this! Current deployment of releases should be based on v{version} tags detected in the GH Actions flow and upload to PyPI. So the GH releases are not automatic and have missed some versions. We had no strong reasons  not to automatise GH releases as well, we were just caring about PyPI and not that much about GH releases. Is this an issue?

**Comment by goanpeca** [comment]
I think this is good for review @jaimergp ?

**Pull request merged**